.travis.yml

```diff
@@ -68,7 +68,7 @@ matrix:
       env: VERSION=2.7 VARIANT=alpine3.9
 
 install:
-  - git clone https://github.com/docker-library/official-images.git ~/official-images
+  - git clone --depth 1https://github.com/docker-library/official-images.git ~/official-images
```

Comment:
Missing space?

Reply:
oh, yes, break in accident when resolving conflicts, thanks!